sass/tji/_data.scss

```diff
@@ -0,0 +1,54 @@
+/*--------------------------------------------------------------
+>>> EXPLORE THE DATA TABLE OF CONTENTS:
+----------------------------------------------------------------
+# GENERAL
+# CHARTS
+
+/*--------------------------------------------------------------
+## CHARTS
+--------------------------------------------------------------*/
+
+#js-TJIChartView {
+  width: 100%;
+  padding: 2em;
+  background-color: #fbfbfb;
+  box-shadow: 2px 3px 6px rgba(64,64,64,0.5);
```

Comment:
This is a cool effect. Usually this type of shadow is used for visual separation, but if the entire chart area is raised, I'm not sure what the separation is for. Would this make more sense applied to individual charts instead?